reference-implementation/include/emitc/types.h

```diff
@@ -100,6 +100,8 @@ struct switch_t<case_t<B, T>> {
 };
 } // namespace detail
 
+/// The elements of a Tensor are stored contiguously in memory in a
+/// column-major layout.
```

Comment:
```suggestion
/// row-major layout.
```

The indexing is done using the row-major layout, i.e. the elements of the last dimension are closest together in memory.

Reply:
I see, changed!